Lib/fontbakery/specifications/ufo_sources_test.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import os
+
+import defcon
+import pytest
+
+from fontbakery.checkrunner import (DEBUG, ERROR, FAIL, INFO, PASS, SKIP, WARN)
+
+check_statuses = (ERROR, FAIL, SKIP, PASS, WARN, INFO, DEBUG)
+
+
+@pytest.fixture
+def empty_ufo_font(tmpdir):
+    ufo = defcon.Font()
+    ufo_path = str(tmpdir.join("empty_font.ufo"))
+    ufo.save(ufo_path)
+    return (ufo, ufo_path)
+
+
+def test_check_ufolint(empty_ufo_font):
+    from fontbakery.specifications.ufo_sources import (
+        com_daltonmaag_check_ufolint as check)
+    _, ufo_path = empty_ufo_font
+
+    print('Test PASS with empty UFO.')
+    c = list(check(ufo_path))
+    status, _ = c[-1]
+    assert status == PASS
```

Comment:
Does ufolint **really** PASS with an empty UFO? Shouldn't it complain about it being empty? And also all other problems that such a degenerated case would likely trigger?

Reply:
I installed the wrong package because the cross-activity on GH distracted me :D

I think ufolint only does a bare minimum checking of technicalities. That's why there are more checks to be made ;)

Reply:
greeeeeeeeeeeeeeeeeeeeeeen

Reply:
Alright !!! :-D